src/litserve/server.py

```diff
@@ -50,6 +50,13 @@
 LONG_TIMEOUT = 100
 
 
+def _inject_meta(context_kwargs: dict, func, *args, **kwargs):
+    sig = inspect.signature(func)
+    if "context_kwargs" in sig.parameters:
```

Comment:
let's call it `context` instead of `context_kwargs` or `meta_kwargs`
let's also change the signature to `_inject_context` accordingly